convert_old_db/convert.cc

```diff
@@ -95,10 +153,12 @@ int main(int argc, char** argv) {
 
     if (!output.empty()) {
       fo << JSON(idxts_t(output_mutation_index++, timestamp)) << '\t' << output << '\n';
+      ++processed_lines;
     }
 
-    ++lines;
+    ++total_lines;
+    last_timestamp = timestamp;
   }
 
-  std::cerr << "Processed " << lines << " lines." << std::endl;
+  std::cerr << "Processed " << processed_lines << " of " << total_lines << " lines." << std::endl;
```

Comment:
$0.02: Number of skipped lines would be useful here. No need to change, just as a way of thinking :)